### PR TITLE
gluon-config-mode-hostname: clarify the use of the nodename

### PR DESCRIPTION
**Pull request by rubo77**
Maybe I can also get the name of the community in a variable? And is there a way to get the ssid here? So I could write this also in the hint.

This is just a first stub, I will enhance this later. also in the english version then.

### CODE REVIEW

**Comment by rubo77** [comment]
I added an english version. about the french, someone with more skill should drom in now

**Comment by rubo77** [comment]
I added an english version. about the french, someone with more skill should drom in now (maybe @Gunstick ?)

**Comment by christf** [comment]
@rotanid what kind of work is it that is still needed?

**Comment by rotanid** [comment]
@christf @rubo77 i added the labels before the last changes were added, i removed them now. please note though, that the final merge decision isn't mine, i just do "1st level" or "front desk" work to help the gluon developers

**Comment by NeoRaider** [comment]
These changes conflict with #885, which is also your PR. Please clean this up or close one of the PRs.

File: docs/site-example/i18n/en.po [hunk 1]
```diff
@@ -13,15 +13,22 @@ msgstr ""
 msgid "gluon-config-mode:welcome"
 msgstr ""
 "Welcome to the setup wizard of your new Freifunk Duckburg node. "
-"Please fill out the following form and submit it."
+"Please fill out the following form and submit it. "
+"The name of this node is the visible name on the Freifunk map. It is different "
+"from the WiFi Network SSID of the Freifunk Network that has a fixed name. "
+"You can choose any name, so think of something descriptive or beautiful."
```
Comment (NeoRaider) [inline docs/site-example/i18n/en.po]: I think this text should rather be added as a help text for the hostname field, not to the welcome text.

**Comment by rubo77** [comment]
I added the text in the hostname package as comments as a stub until i find out, how to add a correct paragraph section there...

**Comment by NeoRaider** [comment]
I must say that I don't like these new texts very much. I don't think they are necessary, either.

**Comment by SimJoSt** [comment]
As we have a lot of non technically well versed users and I hear the question "can I change the name of the network?" I would not dismiss the idea rashly.
But I agree, the text doesn't appeal to me as well. In the next week I could possibly provide my own draft of this.

**Comment by rotanid** [comment]
@SimJoSt , you offered to provide another draft ?
or maybe @rubo77 wants to come up with an improved version?

**Comment by SimJoSt** [comment]
Thank you for pointing that out for me. I entirely forgot about it and didn't have it on my todo list. Now I have.

**Comment by SimJoSt** [comment]
Not that different, but a little shorter and cleaner in it's phrasing:
> Den WLAN-Namen (SSID) welcher dieser Knoten ausstrahlt ist unveränderlich und von Deiner Freifunk-Community festgelegt.  
Der hier festgelegte Name wird für die Karte sowie seine Statusseite verwendet.

Of course a variable to display the SSID would be nice.
I have only submitted a German version, because I wanted to wait for feedback and didn't want to double the work.

**Comment by rotanid** [comment]
thanks! my idea for further improvement:
> Der WLAN-Name (SSID), welchen dieser Knoten ausstrahlt, ist unveränderlich und von deiner Freifunk-Community festgelegt.
> Der hier festgelegte Name wird im Gegensatz dazu für die Knotenkarte sowie die Statusseite des Knotens verwendet.
in the first sentence, there was a grammatical error and missing commas.
in the second sentence, i thought that we have to make the difference clearer with "gegensatz"

**Comment by rubo77** [comment]
I rebased it with the new wording (still untested)

**Comment by rubo77** [comment]
To integrate the ssid from site.conf into the text, we can use something like this here: https://github.com/freifunk-gluon/gluon/pull/930/files#diff-d30fdce48422c5f4e99978f4780634caR15

**Comment by rotanid** [comment]
@rubo77 feel free to do it, if you know a possible solution :-)

**Comment by rotanid** [comment]
@rubo77 do you still plan to finish this or can we close this?

**Comment by rubo77** [comment]
I still plan this, if i find the time

**Comment by rotanid** [comment]
do you plan to work on this significantly, soon? otherwise i would rather close this PR and you open a new one as soon as you want to work on it again...

**Comment by rubo77** [comment]
What is todo here? **I think this could be merged as it is.** 

if someone knows how easily to integrate the ssid from site.conf, feel free to tell us here. but that part could be a new PR also.

**Comment by rotanid** [comment]
last statement by @NeoRaider was:

> I must say that I don't like these new texts very much. I don't think they are necessary, either.

so it's up to him

**Comment by rubo77** [comment]
No, that note was before wwe discussed the texts and changed them completely. the text now should be fine by now.

**Comment by rotanid** [comment]
@NeoRaider central text... your call ;)

**Comment by NeoRaider** [comment]
Hmm... I still don't like these texts. Two concrete issues:
* Gluon texts outside the site configuration must not contain the words "Freifunk" or "community", as Gluon is not used for Freifunk only
* The English version needs to be improved (a WLAN is broadcasted, not "radiated"), and even the German version sounds weird (WLAN wird ausgestrahlt klingt im Passiv besser; anstatt dass ein WLAN-Name ausgestrahlt wird, sollte ein WLAN mit einem Namen ausgestrahlt werden). If the French version is equally clunky (which I can't judge), let's leave the French text empty for now.

How about something like:

"Der Knotenname dient ausschließlich zur Identifikation deines Knotens, z.B. auf einer Knotenkarte. Er hat keinen Einfluss auf den Namen (SSID) des ausgestrahlten WLANs." (also shorter, so more people will read it :smile:)

**Comment by rubo77** [comment]
I also like the new text – much smoother. :)

 I have  adapted everything, even the French (I speak French quite well).

**Comment by rotanid** [comment]
@rubo77 please also change "radiated" to the suggestion neoraider wrote.

**Comment by rubo77** [comment]
> please also change "radiated" to the suggestion neoraider wrote.
done

**Comment by rotanid** [comment]
@rubo77 have you build- and run-tested your current status?
as i don't have much time at the moment, i can't do it myself...

**Comment by rubo77** [comment]
node
in english it works, I build in german now....

**Comment by rubo77** [comment]
img_20180228_180912
works like this!

**Comment by rotanid** [comment]
@rubo77 i don't think it works? it shows the english text although the rest of the page is in german?

**Comment by rubo77** [comment]
it works, but (schande über mein haupt) I uploaded the old screenshot again, wait a minute....

**Comment by rubo77** [comment]
unbenannt

**Comment by rotanid** [comment]
merged @rubo77

**Pull request merged**